samples/Calling/.eslintrc.js

```diff
@@ -49,13 +49,6 @@ module.exports = {
       env: {
         jest: true
       }
-    },
-    {
-      files: ['App.tsx'],
-      rules: {
-        // Allow requiring the package.json
```

Comment:
make sure this is removed from Chat and Meeting samples also

Reply:
Added changes for Chat, no changes required for Meeting.